src/components/PieChart.js

```diff
@@ -0,0 +1,161 @@
+
+import React from 'react';
+import Highcharts from 'highcharts';
+import drilldown from 'highcharts/modules/drilldown.js';
+import noDataToDisplay from 'highcharts/modules/no-data-to-display.js';
+
+function getCategoriesSeriesData(datas) {
```

Comment:
I'd change the name of this from datas to something more specific - maybe analyticsData :)